sites/all/modules/custom/da_malik/da_malik.install

```diff
@@ -19,6 +19,24 @@ function da_malik_enable() {
     module_enable($modules, TRUE);
 	
 }
+
+/**
+ * Implements hook_update_N()
+ * 
+ * Enable debugacademy module and 
+ * revert Theme 2 Blocks Panels Menus feature
+ */
+
+function da_malik_update_7008(&$sandbox) {
+  /*enable new debugacademy module*/
```

Comment:
but if you were writing a comment with /* this notation */, there should be a space after the first * and before the second *. 
so instead of:
/*multi line comment*/
it'd be
/* multi line comment */